explainer.md

```diff
@@ -68,356 +107,405 @@ If the set of pages being badged corresponds to an [Installed Web Application](h
+API](https://www.w3.org/TR/push-api/) and [Periodic Background
+Sync](https://github.com/WICG/BackgroundSync/blob/master/explainer.md#periodic-synchronization-in-design).
+
+### The Push problem
+
+The [Push API](https://www.w3.org/TR/push-api/) allows servers to send messages
+to service workers, which can run JavaScript code even when no foreground page
+is running. Thus, a server push could trigger a `Badge.set`.
+
+However, there is a [de facto standard
+requirement](https://github.com/w3c/push-api/issues/313) that whenever a push is
+received, a notification needs to be displayed. This means it's impossible to
+subtly update a badge in the background while no pages are open, without also
+displaying a notification.
+
+This may be fine for some use cases (if you want to always show a notification
+when updating the badge, possibly with an exception of not showing a
+notification if the app is in the foreground). But we specifically designed this
+API as a *subtle* notice mechanism that does not require a more distracting
+notification in order to set a badge. Because of this de facto requirement, the
+Push API currently isn't suitable for this use case.
+
+There is also a requirement that the user grants notification permission to
+subscribe to push messages.
+
+### Periodic Background Sync
+
+[Periodic Background
+Sync](https://github.com/WICG/BackgroundSync/blob/master/explainer.md#periodic-synchronization-in-design)
+is a proposed extension to the [Background
+Sync](https://wicg.github.io/BackgroundSync/spec/) API, that allows a service
+worker to periodically poll the server, which could be used to get an updated
+status and call `Badge.set`. However, this API is unreliable: the period that it
+gets called is at the discretion of the user agent and can be subject to things
+like battery status.
+
+That means when the page isn't open, you could have the badge indicator update
+every once in awhile, but have no guarantee that it would be up to date.
+
+(Also, the Periodic Background Sync API is not yet implemented.)
+
+It's possible that the use of Push API and Periodic Background Sync together is
+"good enough": high-priority notices (that show a notification) come in through
+a Push and are displayed immediately with a notification; low-priority notices
+are displayed as badges immediately if there are pages open, or "eventually" if
+there are no pages open.
+
+However, in our view, there is a need for *immediate* update to low-priority
+notices: this is akin to the [urgency vs importance](https://en.wikipedia.org/wiki/Time_management#The_Eisenhower_Method)
+dichotomy: a badge is for non-important information but that doesn't mean the
+user doesn't want to see the notice in a timely fashion.
+
+### Possible changes to the Push API
+
+Here we discuss ways to potentially allow the Push API to update badges without
+showing notifications.
+
+**Note: This should not be considered blocking.** The Badge API is perfectly
```

Comment:
Do we have a preference for

> Note: The foo is barred.

vs 

**Note:** The foo is barred

I have no strong feelings but on the off chance you care

```suggestion
> Note: **This should not be considered blocking.** The Badge API is perfectly
```

Reply:
I have a (weak) preference for not using quotation blocks for notes.